pkg/cgroups/cgroups.go

```diff
@@ -527,7 +527,10 @@ func (c *CgroupControl) Stat() (*Metrics, error) {
 	m := Metrics{}
 	for _, h := range handlers {
 		if err := h.Stat(c, &m); err != nil {
-			return nil, err
+			if !os.IsNotExist(errors.Cause(err)) {
+				return nil, err
+			}
+			logrus.Warningf("Failed to Cgroup stats: %v", err)
```

Comment:
```suggestion
			logrus.Warningf("Failed to retrieve Cgroup stats: %v", err)
```
Apologies. That may have come from my previous incomplete suggestion.

Reply:
I think we may need to add some additional logic and error out in case _all_ handlers are gone.